FutureNorthants/northants-website
# Fix error when no signposts in array

Resolves #167 

### Testing
1. Before checking out this branch, edit a service page and go into the Signposting tab. If there is anything in there already, remove it and confirm the removal until no buttons saying 'Remove' are visible. Now click Add council signposting and you'll see there are now two Remove buttons. Click the lower one and confirm the removal. This leaves the page with a signposting entry, but no signposts (you can still see a Remove button that would take out the entire entry - don't click it!). Save and publish. View the page in the front end and you should get a server error.
2. Now checkout this branch. Refresh the page in the frontend. Verify the page now displays.
3. Edit the signposting data again, this time add a signpost link. Leave the URL blank. Refresh the page in the frontend. Verify the page now displays including a signpost link to the council you selected.
4. Edit the signposting data again, this time set the URL field. Refresh the page in the frontend. Verify the page now displays including a signpost link to the URL you entered.
